clients/unshaded/src/main/java/tachyon/client/TachyonFSTestUtils.java

```diff
@@ -39,7 +39,7 @@
    * @throws IOException if <code>path</code> is invalid (e.g., illegal URI)
    */
   public static TachyonFile createByteFile(TachyonFileSystem tfs, String fileName,
-      ClientOptions options, int len) throws IOException {
+      OutStreamOptions options, int len) throws IOException {
```

Comment:
NIT: put `OutStreamOptions` as the last parameter in this signature as others are "required"